translation/pt/patterns/repository-activity-score.md

```diff
@@ -0,0 +1,138 @@
+## Title
+
+Repositório  Pontuação de Atividade
+
+## Patlet
+
+Potenciais contribuidores desejam encontrar projetos InnerSource ativos que precisem de sua ajuda. Ao calcular uma pontuação de atividade do repositório para cada projeto, uma lista classificada de projetos pode ser criada (por exemplo, no [Portal InnerSource](innersource-portal.md)), para que os potenciais contribuidores possam determinar mais facilmente em qual projeto desejam contribuir.
+
+## Problema
+
+**Em que ordem** os projetos InnerSource devem ser apresentados? Indicadores de classificação típicos, como *GitHub Stars*, *Número de Forks*, *Número de Commits*, *Linhas de Código*, *Última Atualização*, não são suficientes para indicar de forma concisa a atividade de um projeto.
+
+Projetos ativos com muita tração, mas também projetos relativamente novos e entusiasmados que precisam de novos contribuidores, devem ser classificados acima de projetos maduros com pouca atividade ou em modo de manutenção.
+
+Uma nova métrica derivada de vários indicadores de desempenho é necessária para definir uma pontuação confiável e versátil para o nível de atividade de um projeto.
+Ela pode ser usada para ordenar projetos de acordo com o nível de atividade deles.
+
+## História
+
+Quando o InnerSource é praticado por um longo período de tempo ou se expande além de um certo número de projetos (digamos 50 para ter um limite significativo), torna-se difícil encontrar os projetos InnerSource mais populares e ativos no momento. Projetos que existem por muito tempo são conhecidos, mas podem não estar tão ativos. Projetos relativamente novos, por outro lado, não têm reputação ou uma comunidade ativa ainda.
+
+Uma lista de projetos InnerSource não deve ser considerada um recurso estático, mas sim um local emocionante para descobrir e explorar projetos novos e ativos, assim como uma página de notícias que lista os tópicos mais interessantes do dia primeiro. Portanto, é benéfico quando a ordem dos projetos é atualizada regularmente e muda de acordo com a popularidade e atividade do projeto.
+
+Essas considerações levaram a um primeiro protótipo para calcular uma pontuação de atividade do repositório, que funcionou surpreendentemente bem e determina uma ordem sempre em mudança de projetos de acordo com a sua atividade.
+
+## Contexto
+
+A descoberta de projetos InnerSource pode ser facilitada com o [Portal InnerSource](innersource-portal.md) e o padrão [Gig Marketplace](gig-marketplace.md), ou promovendo projetos em outros canais de comunicação e plataformas. A pontuação de atividade define uma ordem padrão na qual os projetos são apresentados à comunidade.
+
+## Forces
```

Comment:
```suggestion
## Forças
```

Reply:
Seems like a straightforward fix, as I looked up the word in other patters.
Will commit this :)